### PR TITLE
Make Kubernetes cluster prefixes configurable

### PR DESCRIPTION
**Pull request by marioreggiori**
### Description

Added optional fields to Kubernetes cluster schema for existing prefixes in which the cluster should be deployed.

### Checklist

* [x] added release notes to `Unreleased` section in [CHANGELOG.md](CHANGELOG.md), if user facing change

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

**Comment by marioreggiori** [comment]
Needs `go-anxcloud` release before merge.

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ If the change isn't user-facing but still relevant enough for a changelog entry,
 * (internal)? scope: short description (#pr, @author)
 -->
 
+### Added
+* anxcloud_kubernetes_cluster: optional fields in Kubernetes cluster schema for existing prefixes in which the cluster should be deployed (#123, @marioreggiori)
```
Comment (LittleFox94) [inline CHANGELOG.md:21]: I think we can strip the "in Kubernetes cluster schema" and maybe change "for existing prefixes" to "for using existing prefixes" and "... to deploy the cluster into"

**Comment by marioreggiori** [comment]
The failing test is unrelated (need to configure new VLAN(s) and Prefix(es)).

**Comment by LittleFox94** [comment]
> The failing test is unrelated (need to configure new VLAN(s) and Prefix(es)).

can we get this done before merging this?

**Comment by marioreggiori** [comment]
> > The failing test is unrelated (need to configure new VLAN(s) and Prefix(es)).
> 
> can we get this done before merging this?

Fixed in #125

**Review by LittleFox94 — APPROVED** [review]

**Pull request merged**